docs/install/installing-istio.md

```diff
@@ -216,33 +216,51 @@ helm template --namespace=istio-system \
 
 ```
 
-### Updating your install to use cluster local gateway
-
-If you want your Routes to be visible only inside the cluster, you may want to
-enable [cluster local routes](../serving/cluster-local-route.md). To use
-this feature, add an extra Istio cluster local gateway to your cluster. Enter
-the following command to add the cluster local gateway to an existing Istio
-installation:
-
-```shell
-# Add the extra gateway.
-helm template --namespace=istio-system \
-  --set gateways.custom-gateway.autoscaleMin=1 \
-  --set gateways.custom-gateway.autoscaleMax=2 \
-  --set gateways.custom-gateway.cpu.targetAverageUtilization=60 \
-  --set gateways.custom-gateway.labels.app='cluster-local-gateway' \
-  --set gateways.custom-gateway.labels.istio='cluster-local-gateway' \
-  --set gateways.custom-gateway.type='ClusterIP' \
-  --set gateways.istio-ingressgateway.enabled=false \
-  --set gateways.istio-egressgateway.enabled=false \
-  --set gateways.istio-ilbgateway.enabled=false \
-  install/kubernetes/helm/istio \
-  -f install/kubernetes/helm/istio/example-values/values-istio-gateways.yaml \
-  | sed -e "s/custom-gateway/cluster-local-gateway/g" -e "s/customgateway/clusterlocalgateway/g" \
-  > ./istio-local-gateway.yaml
-
-kubectl apply -f istio-local-gateway.yaml
-```
+### Installing `cluster-local-gateway` for internal cluster-only routes
```

Comment:
I editing this section for clarity and organization. PTAL and ensure its accuracy.